mock/etc/mock/site-defaults.cfg

```diff
@@ -48,6 +48,10 @@
 # None or leaving it undefined uses the default behavior.
 # config_opts['print_main_output'] = None
 #
+# This option sets the beginning of the line with stderr output in build.log file.
+# By default it's an empty string.
+# config_opts['stderr_line_beginning'] = ""
```

Comment:
I would be slightly more inclined to call it `prefix` instead of `beginning`.

Reply:
+1 to `prefix`